Makefile

```diff
@@ -0,0 +1,40 @@
+# Copyright and related rights are licensed under the Solderpad Hardware
+# License, Version 0.51 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+# http://solderpad.org/licenses/SHL-0.51. Unless required by applicable law
+# or agreed to in writing, software, hardware and materials distributed under
+# this License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# Authors:
+# - Thomas Benz <tbenz@iis.ee.ethz.ch>
+
+VSIM        ?= "questa-2022.3 vsim"
+SYNOPSYS_DC ?= "synopsys-2022.03 dc_shell"
```

Comment:
I'd argue we keep agnostic versions here (i.e. only `vsim` and `dc_shell`). If you prefer, we can do default agnostic, but have an automatic switch if the file `/etc/iis.version` (or folder `/usr/sepp`) is found to differentiate internal systems.

Reply:
Done :) I like the check for `/etc/iis.version`.